### DISCUSSION TITLE
require-requests-limits policy - CPU limit ?

### DISCUSSION THREAD

**Discussion by Kostavro** [kyverno/policies#208] [General]
Is it on purpose that there isn't a cpu limit validation in this policy? If not, may I open a small PR to add it?

https://github.com/kyverno/policies/blob/fb012c7d6d541f691b26335d8c78f25bc2bc3496/best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml#L35

**Comment by chipzoller** [comment]
Yes, it's purposefully omitted because setting a CPU limit isn't often a best practice.
